examples/openshift/clamav/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM openshift/origin:latest
+
+# Reset the entrypoint
+ENTRYPOINT ["/bin/sh", "-c"]
+
+RUN yum install -y clamav-server clamav-scanner clamav-data --disablerepo=origin-local-release \
+      && mkdir -p /image-data && chmod 0777 /image-data
```

Comment:
specialized nodes won't account for existing, running containers.  But that also doesn't preclude the use of a job based approach.  I don't necessarily think it's an either/or question in the long run but we probably want to stick with one approach to begin with.